python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -19,14 +19,15 @@
 import urllib.parse
 
 import requests
-from requests.packages.urllib3.util.retry import Retry
+from urllib3.util.retry import Retry
 from threatexchange.exchanges.clients.utils.common import TimeoutHTTPAdapter
 
 
 _DATE_FORMAT_STR = "%Y-%m-%dT%H:%M:%SZ"
 _DEFAULT_ELE = ET.Element("")
 
 T = t.TypeVar("T")
+FEEDBACK = t.List[t.Dict[str, str]]
```

Comment:
blocking: ALL_CAPS usually denotes a constant. In this case, your type is simple enough that you should just inline it.